ExampleApp/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,7 +5,7 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
-  - enabled: 1
-    path: Assets/Scenes/SampleScene.unity
```

Comment:
just asking, what's this change about?

Reply:
I'm not really sure, these changes were automatically by the unity editor, this is also relevant to the changes in sample scene

Reply:
i see it was undone